### DISCUSSION TITLE
Installation of WSL/visual studio code - Part 6, Lesson 3

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#424] [General]
I just finished installing WSL and VScode on my windows computer. Everything seems fine but the button in the bottom left hand corner of my VScode says WSL instead of WSL:Ubuntu like in the video. Does anyone know if that means I did something wrong with the installation?

**Comment by john-lawniczak** [comment]
It's probably in reference to your operating system. Different Linux systems have different outputs. So I doubt you did anything wrong if it all seems to be running properly!
